### ISSUE TITLE
White page when exporting without "theme toggle" button

### ISSUE THREAD

**Issue by RicardoKers** [KosmosisDire/obsidian-webpage-export#256]
**Describe the bug**
I would like to export the html without the side tabs or buttons, when I disable the "Include theme toggle" option the generated HTML only presents a blank page in the browser

**Operating System**
Windows 11

**To Reproduce**
1- I have a smal test vault, I install the plugin and export the HTML files, it works fine, the generated files open in Edge browser and show all information.
2- I Disable the option "Include theme toggle"
3- I repeat the same export process.
4- The generated HTML files show a blank screan (show anithing) whem loaded in Edge browser.

If I open the Generated HTML files in text editor thei are not blank files, they have the code inside.

**Screenshots**
The option disabled and the blak page generated...
<img width="828" alt="Plugin error" src="https://github.com/KosmosisDire/obsidian-webpage-export/assets/49044489/dc613df1-eefc-41d3-a7d9-1bd41a55814f">

### IMPORTANT: Error Log
- The console shows this:
6 plugin:webpage-html-export:56581 failed to find render leaf

- And the Issues tab shows:
Audit usage of navigator.userAgent, navigator.appVersion, and navigator.platform
A page or script is accessing at least one of navigator.userAgent, navigator.appVersion, and navigator.platform. Starting in Chrome 101, the amount of information available in the User Agent string will be reduced.
To fix this issue, replace the usage of navigator.userAgent, navigator.appVersion, and navigator.platform with feature detection, progressive enhancement, or migrate to navigator.userAgentData.
Note that for performance reasons, only the first access to one of the properties is shown.
1 source
codemirror.js:1
Learn more: User-Agent String Reduction

**Comment by KosmosisDire** [comment]
Hi in the next update this will be fixed, I'll let you know when the beta is released. You will also be able to export everything without sidebars or any extra content in the next update as well

**Comment by cremfert** [comment]
Same here, the error that is shown in the console is:
```Unhandled Promise Rejection: TypeError: null is not an object (evaluating 'o.checked=e')```

Thanks for developing this cool export feature - I'm really looking forward to the next version

**Comment by Leiyi548** [comment]
> Hi in the next update this will be fixed, I'll let you know when the beta is released. You will also be able to export everything without sidebars or any extra content in the next update as well
Post update notice

**Comment by KosmosisDire** [comment]
> > Hi in the next update this will be fixed, I'll let you know when the beta is released. You will also be able to export everything without sidebars or any extra content in the next update as well
> > Post update notice

I will update this issue once it is released. I am very busy though so I can't really give a specific timeline. It may be within the week or not until a few moths from now. Not sure,

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
